### ISSUE TITLE
500 server error - timeout

### ISSUE THREAD

**Issue by fearnie85** [gdbarron/VenafiTppPS#151]
Hi All/ Greg,

not really an issue with your PS, but more with long API calls.
Basically venafi timesout if the requests takes longer than 90 seconds. I was wondering if there is a way to perhaps renew the API key somehow.

Or perhaps someone else has had this issue. 

I had raised this here and closed it, but perhaps someone else can help me with this - https://github.com/gdbarron/VenafiTppPS/issues/149

Thanks.

**Comment by gdbarron** [comment]
I don't think renewing the key will help unless it's not one call, but multiple and that would probably cause an authentication error. Otherwise, we would need to increase the time out on the rest call, server side, or possibly both. Do you know for sure that venafi is the one timing out?    If so, the change will need to be done in the web config. I could also show you where you could temporarily add a different timeout to the main rest call for the module as a troubleshooting step.

**Comment by fearnie85** [comment]
yeah that would really help me! I tried to look in the web config, do you know specifically what I can edit in there? I tried to search for something which was closest to what I needed but couldn't find anything.
or is there something I need to add? Any help is much appreciated :)

**Comment by fearnie85** [comment]
I am trying to look into the venafi documentation for configuring the web config, but quite over whelmed with information. any pointers would be mostly appreciated. thanks.
